docs/docsite/rst/user_guide/intro_getting_started.rst

```diff
@@ -108,7 +108,7 @@ In a directory of your choice you can create your first playbook in a file calle
 .. code-block:: yaml
 
     ---
-    - name: My task
+    - name: My playbook
```

Comment:
Please change below as well.

```
  PLAY [My task] ***************************
```

Reply:
Made the additional change you asked for.